crates/orderbook/src/main.rs

```diff
@@ -175,6 +178,19 @@ struct Arguments {
     /// The API endpoint to call the mip v2 solver for price estimation
     #[structopt(long, env)]
     quasimodo_solver_url: Option<Url>,
+
+    /// The maximum time price estimates will be cache for.
+    #[structopt(
+        long,
+        default_value = "10",
```

Comment:
Should we maybe try with lower value than 10s? 
Not sure if applicable in this situation but sometimes we have 3-4 blocks mined in 10s, so ethereum state could change drastically in very short time.

Reply:
I think the reason we see many blocks mined in short duration is the time it takes them to propagate through the network. So in one update of our node it suddenly sees a couple of new blocks. In that situation even caching for 1 second wouldn't help.

Reply:
typo: "cache" -> "cached"